src/app/routes/applications/new-role/new-role.component.ts

```diff
@@ -908,28 +895,11 @@ export class NewRoleComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
-  private _
-
```

Comment:
May I know what is this part?

Reply:
I can't understand myself
as for me something superfluous
![image](https://user-images.githubusercontent.com/82154808/116395515-e4778e80-a82c-11eb-842f-3ecff242322a.png)